components/brave_extension/extension/brave_extension/background.ts

```diff
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import './youtubedown'
```

Comment:
I think this needs to be modularized so we don't just dump everything in background.ts - cc @petemill